deploy/template.yaml

```diff
@@ -91,6 +91,18 @@ parameters:
         "cpu_architecture": "arm64",
         "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-aarch64-live.aarch64.iso",
         "version": "412.86.202212081411-0"
+      },
+      {
+        "openshift_version": "4.13",
+        "cpu_architecture": "ppc64le",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-ppc64le-live.ppc64le.iso",
```

Comment:
there are 4.13 images available, so it's a great opportunity to start using them (for all archs):
https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/latest-4.13/
https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/pre-release/latest-4.13/

Reply:
Done! Switched to using 4.13 dev preview images here and in the corresponding PR for assisted-service.